mkdocs.yml

```diff
@@ -10,9 +10,15 @@ nav:
   - 'Getting started': 'getting_started.md'
   - Usage:
     - 'Running VaSeBuilder': 'usage.md'
-    - 'Parameters': 'parameters.md'
+    - Parameters:
+      - 'BuildValidationSet': 'param_buildvalidationset.md'
+      - 'BuildSpikeIns': 'param_buildspikeins.md'
+      - 'AssembleValidationSet': 'param_assemblevalidationset.md'
+      - 'Argument files': 'argfiles.md'
     - 'Contexts': 'contexts.md'
-    - 'Input files': 'input_file_formats.md'
-    - 'Output files': 'output_files.md'
-      - 'Formats': 'output_file_formats.md'
+    - 'Hashing': 'hashing.md'
+  - Inputs and Outputs:
+    - 'Input Files': 'input_file_formats.md'
+    - 'Output Files': 'output_files.md'
```

Comment:
@MatthieuBeukers Can you check and make sure I didn't mess up the Outputs section? I think I created the Inputs and Outputs section, but just make sure I didn't mess up your new merged output_files.

Reply:
I think this is fine. My mkdocs.yml looks different, but I won't change it so it can be changed properly in your pull-request